include/can/core/ids.hpp

```diff
@@ -125,9 +125,12 @@ enum class ErrorCode {
     invalid_byte_count = 0x3,
     invalid_input = 0x4,
     hardware = 0x5,
+    timeout = 0x6,
+    estop_detected = 0x7,
```

Comment:
question for the future. Are we planning to also have subset error codes that don't apply to every microcontroller (i.e. sensor errors)